types/pnicorp/placePod/uplink.js

```diff
@@ -50,9 +50,29 @@ function consume(event) {
             data.occupancy = 0;
             data.occupied = false;
           }
+
+          // Warm desk 
```

Comment:
possibly try catch this to prevent breaking any existing uplinks?

Reply:
It should not break so no ?

Theres always occupancy available at this point.
The state should be able to be set even if it would not be available. (It then gives out no minutesSinceLastOccupancy anymore)

We also kinda testet it by implementing warm desk logic at some point.